scripts/install_prereqs.sh

```diff
@@ -27,6 +27,14 @@ case "$PROJECT-$PLATFORM-$METHOD" in
     bundle exec pod install --project-directory=Example --repo-update
     bundle exec pod install --project-directory=Functions/Example
     bundle exec pod install --project-directory=GoogleUtilities/Example
+
+    # Set up GoogleService-Info.plist for Storage and Database integration tests. The decrypting
+    # is not supported for pull requests. See https://docs.travis-ci.com/user/encrypting-files/
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+        openssl aes-256-cbc -K $encrypted_2c8d10c8cc1d_key -iv $encrypted_2c8d10c8cc1d_iv \
+            -in GoogleService-Info.plist.enc -out Example/Storage/App/GoogleService-Info.plist -d
```

Comment:
Since this is only useful for Travis, could we put the `GoogleService-Info.plist.enc` somewhere other than the root?